src/Umbraco.Web/RelatedLinksTypeConverter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Umbraco.Core;
+using Umbraco.Web.Models;
+
+namespace Umbraco.Web
+{
+    public class RelatedLinksTypeConverter : TypeConverter
+    {
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            var convertableTypes = new[]
```

Comment:
This should be a static property, you are allocating a new array everytime this executes.